tests/test_servers/test_http_client.py

```diff
@@ -0,0 +1,301 @@
+import json
+import unittest
+from unittest.mock import ANY, MagicMock, Mock, mock_open, patch
+
+import pytest
+
+from runhouse.globals import rns_client
+
+from runhouse.servers.http import HTTPClient
+from runhouse.servers.http.http_utils import pickle_b64
+
+
+class TestHTTPClient(unittest.TestCase):
+    @pytest.fixture(autouse=True)
+    def init_fixtures(self, request):
+        self.base_cluster = request.getfixturevalue("base_cluster")
```

Comment:
unittest.TestCase methods don't support direct fixture injection like plain pytest functions do - since we only really use this fixture for one test (just to use its config_for_rns, not even to directly interact with the cluster since all these client tests are run locally), figured this would an ok workaround